Template/Tag/CustomHtmlTag.web.js

```diff
@@ -125,6 +125,13 @@
                 newScript.setAttribute('type', element.getAttribute('type'));
             }
 
+            // If this script is executing using a nonce, set the nonce on its children too.
+            var scriptWithNonce = TagManager.dom.bySelector('script[nonce]');
+            if (Array.isArray(scriptWithNonce) && scriptWithNonce.length) {
+                scriptWithNonce = scriptWithNonce[0]; // Grab the first element.
+                newScript.nonce = scriptWithNonce.nonce;
```

Comment:
Also this didn't work for me, I checked on chrome browser,
I did this to make it work `newScript.setAttribute('nonce', scriptWithNonce.getAttribute('nonce'));`

Reply:
I can see similar behaviour in Firefox too

Reply:
Interesting. I read that it was better to use the property since the attribute is supposed to be hidden and return and empty string. I tested in Brave (Chromium) and Firefox and it worked correctly in both. I guess I should probably try adding some test cases to see if the tests pass in all of the Travis variations.